src/io.ts

```diff
@@ -208,7 +212,7 @@ export class IO<E, A> {
    * an undefined result. This result is typed as `unknown`, allowing for flexible casting to any
    * desired result type `A` within the `IO` monad's compositional and type-safe framework.
    */
-  static empty = <A>(): IO<unknown, A> => new IO(async () => IO.ok(undefined as any as A));
+  static empty = <A>(): IO<unknown, A> => new IO<unknown, A>(async () => IO.ok(undefined as any as A));
```

Comment:
Wouldn't it be better to return IO<never, A> since it's always an A?

Reply:
Yes, you are correct.